astropy/units/physical.py

```diff
@@ -580,6 +577,7 @@ def get_physical_type(
 # define the physical types
 for unit, physical_type in _units_and_physical_types:
     def_physical_type(unit, physical_type)
+del unit, physical_type
```

Comment:
Is deleting these still necessary with the name change below? If it is, I'd prefer to rename the loop variables to private variables.